### PR TITLE
Improvement in options page fixes #35 #15

### LINKED ISSUE #35: Redesign the popup sheet
Redesign the popup sheet to make it look something like this - 
image

The settings button would redirect the user to the options page similar to the current options button.

**Resources**
- Use this [mockup](https://www.figma.com/file/GGHxsmNS9AhOq4Y1Co9wkT/Untitled?node-id=0%3A1) for detailed styles. 
- Get the settings icon from [here](https://feathericons.com/?query=settings)

### PR DESCRIPTION
**Pull request by fabcodingzest**
hey @prateek3255, I am done! Will be waiting for your feedback tomorrow :)

- **What does this PR do?**
This PR styles the popup and options page as required in the mockup of #35 issue #9 

- **Any background context you want to provide?**
I have changed the Javascript files a bit but I made sure everything works as before, you might wanna check it out locally.
I really hope you like it, Thank you for assigning me the work.

- **Screenshots and/or Live Demo**

### CODE REVIEW

File: popup.css [hunk 1]
```diff
@@ -1,112 +1,98 @@
...
-}
-
-.btn.red {
-  background-color: #e74c3c;
-}
-
-.btn.red:hover,
-.btn.red:focus {
-  background-color: #c0392b;
-}
```
Comment (prateek3255) [inline popup.css]: @fabcodingzest Your PR has too many changes in the files irrelevant to the issue, like why did you change all these styles from popup.css.

Comment (fabcodingzest) [reply popup.css]: Oh I changed the popup design as you asked in issue #35 so I made my other classes according to how I changed the markup for popup a bit to use flexbox. I pretty much added 6 files I guess 3 for options nd 3 for popup - it runs fine thou as I tried it locally from my branch .

Tell me if u need any changes i would make them I hope.

Comment (fabcodingzest) [reply popup.css]: Hey @prateek3255 I am waiting for your feedback ! 😅
